shapers/base.lua

```diff
@@ -67,8 +67,18 @@ function shaper:measureChar (char)
    local options = SILE.font.loadDefaults({})
    options.tracking = SILE.settings:get("shaper.tracking")
    local items = self:shapeToken(char, options)
-   if #items > 0 then
-      return { height = items[1].height, width = items[1].width, depth = items[1].depth }
+   SU.dump({ char, items })
```

Comment:
This is going to be very verbose without any real need for it, no?

Reply:
Um. Yes of course. That's just debug output leftover from development and should not have been in the commit at all.